README.md

```diff
@@ -30,9 +30,14 @@ $ curl -u joel:joel1234 -sS http://localhost:5000/users -X GET
 Example from AngularJS:
 
 ```
+import { Http, Response, Headers } from '@angular/http'
+
```

Comment:
Sorry @ifoche, not sure what I saw... Definitely is not a good idea to work on Sunday!